### DISCUSSION TITLE
battery limiter not working

### DISCUSSION THREAD

**Discussion by RoyZafrir** [seerge/g-helper#1800] [General]
hi everyone,
just tried this tool for the first time .. 
I went through the steps as described but it seems like the battery charge limit not working no matter what i do . 
i opened log file and it say "batterylimit : 80 = ok" (something like that), which means it suppose to work. (i removed MyAsus).

any help is appreciated 
thank you :)

**Comment by seerge** [comment]
@RoyZafrir what do you understand under "not working" ? 

Check corresponding [section in FAQ](https://github.com/seerge/g-helper?tab=readme-ov-file#battery-charge-limiter-is-not-working)

**Reply by RoyZafrir** [reply]
i tired all % options , 50-90 nothing working and no asus services are on at the moment

**Reply by seerge** [reply]
Can you define "not working" ?
